libc/src/__support/CPP/limits.h

```diff
@@ -36,41 +36,53 @@ template <> class numeric_limits<int> {
 public:
   LIBC_INLINE static constexpr int max() { return INT_MAX; }
   LIBC_INLINE static constexpr int min() { return INT_MIN; }
+  LIBC_INLINE_VAR static constexpr int digits = CHAR_BIT * sizeof(int) - 1;
```

Comment:
Refactor all these to a `consteval int digits_for<T>()` ? Or maybe even:

```
template <> class numeric_limits<int> : public numeric_limits_impl<int, INT_MAX, INT_MIN> {};
```

and derive everything in `numeric_limits_impl` ?